### DISCUSSION TITLE
Two (or more) port - one app

### DISCUSSION THREAD

**Discussion by HighError** [caprover/caprover#1861] [Q&A]
Is it possible to configure the application to use two domains for two ports? For example auth.example.com for 3001 and admin.auth.example.com for 3002. These two ports/domains are required for the [logto](https://logto.io/) application

**Comment by githubsaturn** [comment]
Yes! There is a workaround for that! 

There is an app in the one click app list called "Nginx Reverse Proxy". Deploy that and use `srv-captain--yourappname:3002` as the upstream address!
